index.html

```diff
@@ -41,12 +41,114 @@ <h1>Emmanuel Mensah</h1>
                         and contact me.
                     </p>
                 </div>
-                <div class="text_button">
+                <div class="text_div">
                     <p>Start Collaboration</p> 
                 </div>
-
                 <div class="mobile_image"></div>
             </section>  
+                <!-- work section  -->
+            <section class="works_section">
+                <div class="project_text text_color text_header_font">
+                    <h2>Projects</h2>
+                </div>
+                <div class="work_container">
+                    <div class="grid_1 grid_picture"></div>
+                    <div class="grid_row">
+                        <p class="pro_text text_body_font">Tonic</p>
+                        <ul class="list_category text_body_font">
+                            <li>Html</li>
+                            <li>CSS</li>
+                            <li>Javascript</li>
+                        </ul>
+                        <div class="project_click_class text_body_font">
+                           <p> See this project </p>

```

Comment:
- [ ] Kindly make `See this project` a link a or button tag, and follow the button styling: Active, Hover and Pressed (as seen in the design)

Reply:
@Cathella,please kindly assist in pressed CSS styling

Reply:
I think you can just use the current styles and apply them to a button or link instead